app/views/school/la_funded_places/laptop_types.html.erb

```diff
@@ -0,0 +1,40 @@
+<%- title = t('page_titles.iss.laptop_types') %>
+<%- content_for :title, title %>
+<%- content_for :before_content do %>
+  <%- school_breadcrumbs items: [ { t('page_titles.iss.order_devices') => order_devices_school_path(@school)}, title], school: @school, user: impersonated_or_current_user %>
+<%- end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= title %>
+    </h1>
+    <p class="govuk-body">
+      We recommend checking with schools, families or pupils as soon as possible to find out which device type and IT settings would be most suitable.
+    </p>
+    <p class="govuk-body">
+      Depending on availability, you can choose from:
+    </p>
+    <h2 class="govuk-heading-s">
+      Standard Windows laptops
+    </h2>
+    <p class="govuk-body">
+      You’ll be responsible for arranging and maintaining safeguarding measures, like content filtering.
+    </p>
+    <h2 class="govuk-heading-s">
+      DfE Restricted Windows laptops
+    </h2>
+    <p class="govuk-body">
+      These have temporary safeguarding settings installed — content filtering and mobile device management software — which you’ll need to replace when they expire on 30 September 2021. You’ll need to reset these devices before you can apply new settings, or download some types of software.
+    </p>
+    <h2 class="govuk-heading-s">
+      Google Chromebooks
+    </h2>
+    <p class="govuk-body">
+      You’ll need to provide us with the relevant school’s email domain for <span class="app-no-wrap">G Suite for Education</span>, and a recovery email address.
```

Comment:
... email domain for Google Workspace for Education Fundamentals (formerly G Suite for Education), and a recovery ...

Reply:
Done